web/app/fr/gospeak/web/auth/services/AuthSrv.scala

```diff
@@ -110,18 +112,65 @@ class AuthSrv(userRepo: AuthUserRepo,
     } yield authUser
   }
 
+  def createOrEdit(socialProfile: CommonSocialProfile): IO[AuthUser] = {
+    def create(login: Login): IO[AuthUser] = {
+      val now = Instant.now()
+      for {
+        data <- SocialProfile.from(socialProfile).toUserData.toIO
+        u <- userRepo.create(data, now)
```

Comment:
It's probably better to add a `create(p: SocialProfile)` in `UserRepo`, so it can:
- check the slug availability and change it if needed
- add more infos in the `User.Profile` if available
- set the `emailValidated` to Some

The `createAccountValidationRequest` and `validateAccount` are not required